EpsilonCCS/Src/Data/BatteryData.c

```diff
@@ -0,0 +1,3 @@
+#include "BatteryData.h"
+
+struct BatteryData batteryData = {};
```

Comment:
Not sure if this matters much but this initializer form appears to be a gnu extension supported in only a few compilers and it is not standard C. It is however standard C++. Refer to [this](http://en.cppreference.com/w/c/language/struct_initialization) or [this](http://stackoverflow.com/questions/11152160/initializing-a-struct-to-0).  Might be better to change it to `{0}`